htdocs/mrp/class/mo.class.php

```diff
@@ -1125,7 +1125,20 @@ public function getNomUrl($withpicto = 0, $option = '', $notooltip = 0, $morecss
 				$label = $langs->trans("ShowMo");
 				$linkclose .= ' alt="'.dol_escape_htmltag($label, 1).'"';
 			}
-			$linkclose .= ' title="'.dol_escape_htmltag($label, 1).'"';
+			$labelProd = '';
+			$subProd = new Product($this->db);
+			$resSub = $subProd->fetch($this->fk_product);
+			if ($resSub) {
+				$labelProd = '<br>'.$langs->trans("Product").' : ' . $subProd->ref;
+			}
+
+
+
+			$title  = dol_escape_htmltag($label, 1);
+			$title .= $labelProd;
+
+
+			$linkclose .= ' title="'.$title.'"';
```

Comment:
The dol_escape_htmltag() must be here inside the title=" "
Otherwise, some part of the string is not encoded (like the $subProd->ref).